stylus/settings/themes/inverted.styl

```diff
@@ -1,16 +1,13 @@
-@require 'settings/palette.styl'
-palette=json('../../theme/palette.json', { hash: true })
 
-$translucentWhite1=rgba(255, 255, 255, 0.24)
-$translucentWhite2=rgba(255, 255, 255, 0.88)
-$translucentWhite3=rgba(255, 255, 255, 0.16)
+palette=json('../../../theme/palette.json', { hash: true })
 
-/*
-root overrides are only for depracted variables.
-*/
+// @stylint off
```

Comment:
I'm curious, what lint errors are you disabling here?

Reply:
C'est le linter stylus, qui n'est pas toujours super approprié. L'idée serait probablement de le rendre plus pertinent plutôt que de le couper 👍 Pour gagner du temps je me suis appuyé sur ce qui était déjà fait